config/config-api/src/com/thoughtworks/go/config/PipelineConfig.java

```diff
@@ -164,7 +168,11 @@ private void validateStageNameUniqueness() {
 
     private void validateLabelTemplate() {
         if (XmlUtils.doesNotMatchUsingXsdRegex(LABEL_TEMPLATE_FORMAT_REGEX, labelTemplate)) {
-            addError("labelTemplate", "Invalid label. Label should be composed of alphanumeric text, it should contain material revision or ${COUNT}, or use params as #{<param-name>}.");
+            addError("labelTemplate",
```

Comment:
I like the message which is used in TOOLTIP_LABEL_TEMPLATE. 
Do you suggest to return TOOLTIP_LABEL_TEMPLATE as localized error message?
For me it looks like as the localized messages are in the server module, so not available for the config-api or common.